diff --git a/Makefile.libretro b/Makefile.libretro_c
index 8046706..ddf701d 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro_c
@@ -143,10 +143,10 @@ else ifeq ($(platform), miyoo)
 
 # (armv7 a7, hard point, neon based) ### 
 # NESC, SNESC, C64 mini 
-else ifeq ($(platform), classic_armv7_a7)
+else ifeq ($(platform),$(filter $(platform),classic_armv7_a7 unix-armv7-hardfloat-neon))
 	TARGET := $(TARGET_NAME)_libretro.so
 	fpic := -fPIC
-	SHARED := -shared -Wl,-version-script=link.T -Wl,-no-undefined
+	SHARED := -shared -Wl,-version-script=link.T -Wl,-no-undefined -fPIC
 	PLATFORM_DEFINES += -Ofast \
 	-flto=4 -fwhole-program -fuse-linker-plugin \
 	-fdata-sections -ffunction-sections -Wl,--gc-sections \
